### PR TITLE
SPOPT-1562

### PR DESCRIPTION
**Pull request by mpundlik**
Added resources.go to separate out resource from schema model.

### CODE REVIEW

File: cmd/cassandra/handlerHelper.go [hunk 1]
```diff
@@ -16,97 +16,62 @@ package main
...
 	}
 
 	return nil
 }
 
 //getAllTweets used to handle GET to get all tweets from cassandra database
-func getAllTweets(db sql.Broker) (result *[]tweet, err error) {
+func getAllTweets(db sql.Broker) (result *[]tweetResource, err error) {
+
+	var TweetTable = &tweet{}
```
Comment (jozef-slezak) [inline cmd/cassandra/handlerHelper.go]: Please do not make multiple for "table" object

File: cmd/cassandra/handlerHelper.go [hunk 2]
```diff
@@ -16,97 +16,62 @@ package main
...
-
-	return result, nil
-}
+	var TweetTable = &tweet{}
+	tweet := &tweetResource{}
 
-//deleteTweetByID used to handle DELETE to delete a tweet from cassandra database
-func deleteTweetByID(db sql.Broker, id string) (err error) {
+	query1 := sql.FROM(TweetTable, sql.WHERE(sql.Field(&TweetTable.ID, sql.EQ(id))))
+	db.GetValue(query1, tweet)
```
Comment (jozef-slezak) [inline cmd/cassandra/handlerHelper.go]: please, check the results of GetValue

File: cmd/cassandra/handlerHelper.go [hunk 3]
```diff
@@ -16,97 +16,62 @@ package main
...
-	phoneMap1 := map[string]phone{"Home": homePhone, "Cell": cellPhone}
-	homeAddr := address{City: "San Jose", Street: "123 Tasman Drive", Zip: "95135", Phones: phoneMap1}
-
-	phoneMap2 := map[string]phone{"Work": workPhone}
-	workAddr := address{City: "San Jose", Street: "255 E Tasman Drive", Zip: "95134", Phones: phoneMap2}
+//deleteTweetByID used to handle DELETE to delete a tweet from cassandra database
+func deleteTweetByID(db sql.Broker, id string) (err error) {
 
-	addressMap := map[string]address{"Home": homeAddr, "Work": workAddr}
+	var TweetTable = &tweet{}
```
Comment (jozef-slezak) [inline cmd/cassandra/handlerHelper.go]: same conmment releated to table as above...

File: cmd/cassandra/handlerHelper.go [hunk 4]
```diff
@@ -16,97 +16,62 @@ package main
...
 
-	var insertUser1 = &user{ID: "user1", Addresses: addressMap}
-	insertErr1 := insertUserTable(db, insertUser1)
-	if insertErr1 != nil {
-		return insertErr1
-	}
+	query1 := sql.FROM(TweetTable, sql.WHERE(sql.Field(&TweetTable.ID, sql.EQ(id))))
+	err = db.Delete(query1)
```
Comment (jozef-slezak) [inline cmd/cassandra/handlerHelper.go]: maybe just inline the query

File: cmd/cassandra/handlerHelper.go (hunk 1 above)
Comment (mpundlik) [reply cmd/cassandra/handlerHelper.go]: done

File: cmd/cassandra/handlerHelper.go (hunk 2 above)
Comment (mpundlik) [reply cmd/cassandra/handlerHelper.go]: done

File: cmd/cassandra/handlerHelper.go (hunk 3 above)
Comment (mpundlik) [reply cmd/cassandra/handlerHelper.go]: done

File: cmd/cassandra/handlerHelper.go (hunk 4 above)
Comment (mpundlik) [reply cmd/cassandra/handlerHelper.go]: done

**Pull request merged**